Build-Dependencies.ps1

```diff
@@ -114,34 +116,25 @@ function Run-Stages {
 }
 
 function Package-Dependencies {
-    if ( $script:PackageName -like 'qt*' ) {
-        $ArchiveFileName = "windows-deps-${PackageName}-${CurrentDate}-${Target}-${Configuration}.zip"
-    } else {
-        $ArchiveFileName = "windows-${PackageName}-${CurrentDate}-${Target}.zip"
-    }
-
-    Push-Location -Stack BuildTemp
-    Set-Location $ConfigData.OutputPath
+    Push-Location -Stack BuildTemp -Path $ConfigData.OutputPath
 
     Log-Information "Cleanup unnecessary files"
-    $Items = @(
-        "./bin/bison.exe"
-        "./bin/libiconv2.dll"
-        "./bin/libintl3.dll"
-        "./bin/m4.exe"
-        "./bin/pcre2*"
-        "./bin/regex2.dll"
-        "./cmake/pcre2*"
-        "./include/pcre2*"
-        "./lib/pcre2*"
-        "./lib/pkgconfig/libpcre2*"
-        "./man1/pcre2*"
-        "./man3/pcre2*"
-        "./share/bison"
-        "./share/doc/pcre2"
-    )
-
-    Remove-Item -ErrorAction 'SilentlyContinue' -Path $Items -Force -Recurse
+
+    switch ( $PackageName ) {
+        dependencies {
+            Get-ChildItem ./bin/*.lib | Move-Item -Destination ./lib
+            Get-ChildItem ./bin/* -Exclude 'libcurl.dll','lua51.*,','Lib','swig.exe' | Remove-Item
+            Get-ChildItem ./cmake/pcre2*,./lib/pcre2* | Remove-Item
+            Remove-Item -Recurse ./lib/pkgconfig
+            Remove-Item -Recurse ./man
+            Get-ChildItem ./share/* -Exclude 'nlohmann?json*' | Remove-Item -Recurse
```

Comment:
Is the single character wildcard intentional?

Reply:
Correct, nothing apart from the nlohmann files is needed in `shared` (Swig's library files on Windows are always put in a directory called `Libs` adjacent to the binary).

Reply:
("nlohmann files" consists of CMake and pkg-config files, I didn't bother to break that down further to also remove the pkg-config directory here).

Reply:
I meant the specific format of `nlohmann?json*` versus `nlohmann_json*`. Are there files that do not match the latter? I could only find files that match the latter more explicit pattern in the current deps package.

Reply:
Cannot really say after all these months - I _guess_ this might have been because of a mismatch between the dependency name "nlohmann-json" and the generated file names based on "nlohmann_json", but you're right that the generated files seem to follow the latter pattern only.